src/instructlab/sdg/utils/chunking.py

```diff
@@ -21,12 +26,24 @@ def chunk_document(documents: List, server_ctx_size, chunk_word_count) -> List[s
     """
     Iterates over the documents and splits them into chunks based on the word count provided by the user.
     Args:
-        documents (dict): List of documents retrieved from git (can also consist of a single document).
+        documents (list): List of documents retrieved from git (can also consist of a single document).
         server_ctx_size (int): Context window size of server.
         chunk_word_count (int): Maximum number of words to chunk a document.
     Returns:
          List[str]: List of chunked documents.
     """
+
+    # Checks for input type error
+    if isinstance(documents, str):
+        documents = [documents]
+        logger.info(
```

Comment:
I would change this to `logger.debug` since you're providing details that only programmers would understand. Imagine `INFO` as the level of information you'd want to see when you're running a program and you don't know how it works internally.

Reply:
ah, yeah, that's what I had suggested originally. I can submit a follow-up PR for it

Reply:
As mentioned above.